src/pybind/mgr/volumes/fs/async_job.py

```diff
@@ -118,10 +118,16 @@ class AsyncJobs(threading.Thread):
     def __init__(self, volume_client, name_pfx, nr_concurrent_jobs):
         threading.Thread.__init__(self, name="{0}.tick".format(name_pfx))
         self.vc = volume_client
-        # queue of volumes for starting async jobs
+        # deque where each member is a string and where each string is name of
+        # volume for which async jobs needs to be started.
         self.q = deque()  # type: deque
-        # volume => job tracking
+
+        # self.jobs is a dictionary where string is the key and tuple is the
+        # value. each a string is volume name and each the tuple will contain
+        # two members: the async job and thread instance that performs that job.
+        # inshort, {volname: job tracking}.
```

Comment:
self.jobs is a dictionary of `volname:(async job, thread instance)` pairs.

Reply:
Updated the comment. Same here too: I've kept it a bit verbose so that no scope for guessing is left for future readers.